lesson0.md

```diff
@@ -1,155 +1,55 @@
-
-So GPL "coerces" Open Source on anything that uses it whereas MIT is more flexible. There are good reasons for using both and many amongst the WordPress core developers are very committed to the coercive nature of GPL. But many big companies are suspicious of it.
-
-If you wish to find out more about Free Software, Open Source and licensing, please watch the excellent documentary ["Revolution OS"](https://www.youtube.com/watch?v=4vW62KqKJ5A) on YouTube.
-
-
-These sections have been sourced from:
-* [GNU - Free Software Definition](https://www.gnu.org/philosophy/free-sw.html)
-* [GNU - What is Copyleft?](https://www.gnu.org/copyleft/copyleft.html)
-* [Make WordPress Training - What is Open Source](https://make.wordpress.org/training/handbook/user-lessons/what-is-open-source/)
-* [Opensource.com - What's the difference between open source software and free software?](https://opensource.com/article/17/11/open-source-or-free-software)
+The WordPress module is four weeks of classes designed to get you up and running with WordPress and learn it's various features so that you can work with your fellow students to build a WordPress site. You will use use your existing HTML, CSS and JavaScript knowledge to create a WordPress theme and plugin and we will explore the WordPress Rest API to write a small React web app. 
 
+In preperation for the first class please read the following and try and complete the exercises. If you have any problems please post a message on Slack.
 
-## What is WordPress? WordPress.com versus WordPress.org
+**What we will learn today?**
+- [What is WordPress?](#what-is-wordpress)
+- [WordPress.com versus WordPress.org](#wordpresscom-versus-wordpressorg)
+- [Set up your WordPress development environment](#set-up-your-wordpress-development-environment)
+- [Homework](#homework)
+- [Resources](#resources)
 
-WordPress is an Open Source Content Management System (CMS), released in 2003 by Matt Mullenweg and Mike Little. As of 2017, it has 59% of the CMS marketshare and 28% of all websites, worldwide.
+---
 
-Although WordPress started as a blogging platform, it can also be used for building websites (large and small), e-commerce sites (with the WooCommerce plugin), private community platforms (with the BuddyPress plugin), forums (with the bbPress plugin), etc.
+## What is WordPress?
 
-Some examples of these could be: [Kinning Park Complex](http://www.kinningparkcomplex.org/), [Sony Music](https://www.sonymusic.com/), [Duracell Lighting](http://www.duracelllighting.com/), and [The Rolling Stones](http://www.rollingstones.com/)
+WordPress is an Open Source Content Management System (CMS), released in 2003 by Matt Mullenweg and Mike Little. It was released in 2003 by Matt Mullenweg and Mike Little. As of 2017, it has 59% of the CMS marketshare and 28% of all websites, worldwide. It is:
```

Comment:
`released in 2003 by MW and ML` is mentioned twice.
Also, we could update the stats to the 2018 numbers, i.e. 30%